vmm/src/lib.rs

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 //
+#[macro_use]
```

Comment:
Leave one blank line between the License header and the first line of code.

Reply:
Done!